docs/content/direct/examples.md

```diff
@@ -70,12 +79,14 @@ manager which connects to the `wds1` front-end and the `imbs1` OCM control plane
 1. Run the OCM based transport controller in a pod.  
 **NOTE**: This is work in progress, in the future the controller will be deployed through a Helm chart.
 
-   Run [transport deployment script](../../../scripts/deploy-transport-controller.sh).  
-   This script expects to get three arguments - (1) wds name; (2) imbs name; and (3) transport controller image.  
+   Run [transport deployment script](../../../scripts/deploy-transport-controller.sh), as follows.
+   This script requires that the user's current kubeconfig context be for the kubeflex hosting cluster.
+   This script expects to get two or three arguments - (1) wds name; (2) imbs name; and (3) transport controller image.  
```

Comment:
Why do we need the lengthy description why not put the command itself + explanation below of the parameters (like a help output of a command)

Reply:
I think it is better to lead with an explanation of what is going on. People often execute displayed commands as soon as they are seen, so I think it is helpful to give the understanding first.